assets/scss/modules/_buttons.scss

```diff
@@ -47,4 +53,44 @@
     }
   }
 }
+
+.button-loading {
```

Comment:
i think this should be button--loading

Reply:
Gracias @mvillaseco tengo que hacer otros cambios de clase con doble dash, caundo haga ese refactor incluyo esto

Reply:
Estamos tratando de usar clases BEM siempre que sea posible `block-element--modifier` seria la sintaxis por eso `button--loading`

Reply:
Creo que en general no tenemos `--`, conviene cambiarlo solo acá por ahora o luego hacer un PR para cambiarlo en todos lados?

Reply:
Acá me dicen que en algunos lugares ya se está usando `--`, entonces me parece buena idea aplicarlo también acá

Reply:
Solo para esta clase o las que se agreguen en este PR.

Reply:
En doppler y relay lo estabamos manejando asi.